src/option.cpp

```diff
@@ -392,6 +389,20 @@ bool process_option_line_compat_0_74(const std::string &cmd,
    return(false);
 } // process_option_line_compat_0_74
 
+
+bool process_option_line_compat_0_75(const std::string &cmd, const char *filename)
+{
+   if (cmd == "pp_space")
+   {
+      OptionWarning w{ filename, OptionWarning::MINOR };
+      w("option '%s' is deprecated; did you want to use '%s' instead?",
+        cmd.c_str(), options::pp_space_after.name());
```

Comment:
and pp_space_before ??

Reply:
'pp_space' has been renamed into 'pp_space_after' and that is what is needed to preserve the same behavior. 'pp_space_before' offers additional functionality only.
Nevertheless I have amended the message to inform the user of the new options.